src/harness/testcases/WINNF_FT_S_BPR_testcase.py

```diff
@@ -0,0 +1,312 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import logging
+import os
+
+from reference_models.antenna import antenna
+from reference_models.geo import utils
+from reference_models.propagation import wf_itm
+
+import sas
+import sas_testcase
+from util import configurable_testcase, writeConfig, loadConfig
+
+
+class BorderProtectionTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_BPR_2_default_config(self, filename):
+    """Generates the WinnForum configuration for BPR.2."""
+
+    # CBSD 1: Category A CBSD within 56 km of the Canadian border with
+    # omni-directional antenna.
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_1['installationParam']['latitude'] = 48.997796
+    device_1['installationParam']['longitude'] = -100.958252
```

Comment:
Device 1 is close to the border, but seems located in a river, is it a minor concern ?

Reply:
Thanks, Changed it away from the river :)